extension/src/aggregate_builder_tests.rs

```diff
@@ -97,12 +88,14 @@ mod tests {
     #[pg_test]
     fn test_anything_in_experimental_and_returns_first() {
         Spi::execute(|client| {
-            let output = client.select(
-                "SELECT toolkit_experimental.anything(val) \
+            let output = client
+                .select(
+                    "SELECT toolkit_experimental.anything(val) \
```

Comment:
I don't know if it's worth doing (in this PR or at all) but this is one of SEVERAL places where `rustfmt` makes the SQL statement inside the string have a peculiar alignment.

Reply:
I think the answer is to use raw string literals and align manually.  We have some of those already, as well as some use of raw that still doesn't have clean alignment.

I think we should isolate rustfmt changes to one commit, so if Thomas does feel like doing anything about this right now, let's do it before or after this.

But I think it can just wait.

Reply:
Along these lines, I suggest NOT squashing this, but keeping the three commits you have now.